ddtrace/contrib/grpc/client_interceptor.py

```diff
@@ -46,37 +46,52 @@ class _ClientCallDetails(
     pass
 
 
-def _handle_response_or_error(span, response_or_error):
-    # response_of_error should be a grpc.Future and so we expect to have
+def _future_done_callback(span):
+    def func(response):
+        try:
+            # pull out status code from gRPC response to use both for `grpc.status.code`
+            # tag and the error type tag if the response is an exception
+            status_code = to_unicode(response.code())
+            span.set_tag(constants.GRPC_STATUS_CODE_KEY, status_code)
+
+            if status_code != grpc.StatusCode.OK:
```

Comment:
Sounds like this is always `True` since you converted `status_code` to unicode?
Even if that works, that looks suspicious.

Reply:
@jd the perils of copy and pasting code 🤦‍♂ thanks!